src/test/java/com/synopsys/integration/detect/battery/docker/util/DetectDockerTestRunner.java

```diff
@@ -50,17 +50,18 @@ private DockerDetectResult runContainer(DetectCommandBuilder detectCommandBuilde
                 dockerImageProvider.installImage(imageName, dockerClient);
                 Assertions.assertTrue(detectDockerRunner.imageExists(imageName, dockerClient), "Image provider was unable to install the image. Not sure how to help :(");
             }
-
+        } catch (Exception e) {
+            Assertions.assertNull(e, "An exception occurred while building the docker file! ");
+        }
```

Comment:
Separated exception handling for the "docker build" stage from the "docker run" stage to help identify the cause of failure in future.